src/ecrc-frontend/src/components/base/sideCard/SideCard.css

```diff
@@ -94,13 +94,13 @@
 
 .dashboard-spacing {
   padding-left: 50px;
-  padding-right: 50px;
+  /* padding-right: 50px; */
```

Comment:
nit: remove if not required

Reply:
same in one other place as well

Reply:
Good catch! Was making sure it didn't break anywhere and then forgot about them.

Reply:
Fixed